package.json

```diff
@@ -31,21 +31,20 @@
   ],
   "scripts": {
     "dev": "gulp server",
-    "test": "gulp lint:scripts"
+    "test": "gulp lint:scripts",
+    "build": "web-ext build --ignore-files=dev/ docs/ Extensions/ Themes/ .* gulpfile.js LICENSE package.json package-lock.json *.md *.sh"
```

Comment:
It would be nice to have a more robust way to specify which files to ignore but I'm guessing this is as good as it gets for now

Reply:
yeah, sadly the only alternative way to pass these in is environment variables, which we could specify easily in github actions... but not so easily with just npm, and making local dev harder seems like a bad idea